tests/test_admin.py

```diff
@@ -2119,6 +2119,35 @@ def test_changelist_view_redirects_on_url_params_that_dont_exist(self):
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, expected_redirect)
 
+    def test_changelist_view_requires_change_permission(self):
```

Comment:
Is there a test for when the permission has been added? I think all the other changelist views work on superuser, so there should probably be a test that checks this for a regular staff user with the change permission.